policy/modules/apps/cdrecord.te

```diff
@@ -113,3 +132,98 @@ tunable_policy(`use_nfs_home_dirs',`
 optional_policy(`
 	resmgr_stream_connect(cdrecord_t)
 ')
+
+########################################
+#
+# Cdripper (e.g. cdda2wav) local policy
+#
+
+allow cdripper_t self:capability { ipc_lock sys_nice setgid setuid dac_override sys_rawio };
+allow cdripper_t self:process { getcap getsched setcap setrlimit setsched sigkill };
+allow cdripper_t self:fifo_file rw_fifo_file_perms;
+allow cdripper_t self:tcp_socket create_socket_perms;
+allow cdripper_t self:udp_socket create_socket_perms;
+allow cdripper_t cdripper_exec_t:file execute_no_trans;
+
+corecmd_exec_bin(cdripper_t)
+corecmd_exec_shell(cdripper_t)
+
+corenet_tcp_connect_all_unreserved_ports(cdripper_t)
+
+dev_list_all_dev_nodes(cdripper_t)
+dev_read_sysfs(cdripper_t)
+
+domain_interactive_fd(cdripper_t)
```

Comment:
What made you add this? This interface is very broad and probably should not be used here

Reply:
ACK. Removed the unneeded interface calls (inherited from the existing cdrecord local policy).